cime/scripts/Tools/Makefile

```diff
@@ -306,13 +289,9 @@ ifeq ($(strip $(USE_ALBANY)), TRUE)
 endif
 
 ifeq ($(strip $(USE_KOKKOS)), TRUE)
-  ifdef KOKKOS_PATH
-    INC_KOKKOS:=$(KOKKOS_PATH)/include
-    LIB_KOKKOS:=$(KOKKOS_PATH)/lib
-  else
-    $(error $(HOMME_TARGET) requires env. variable KOKKOS_ROOT to be set)
-  endif
-	CPPDEFS += -DUSE_KOKKOS_KERNELS
+  CPPDEFS += -DUSE_KOKKOS_KERNELS
```

Comment:
I think USE_KOKKOS_KERNELS is not needed. The xx code already defines it if not present. It was mostly needed in the cmake logic, to establish linking stuff. We can remove it now.